Ardupilog.m

```diff
@@ -278,13 +277,6 @@ function delete(obj)
                 newFmt = char(trimTail(msgData(7:22)));
                 newLabels = char(trimTail(msgData(23:86)));
                 
-                % Create dynamic property of Ardupilog with newName, and add to logMsgGroups array
-                if isempty(obj.logMsgGroups)
-                    obj.logMsgGroups = addprop(obj, newName);
-                else
-                    obj.logMsgGroups(end+1) = addprop(obj, newName);
-                end
-                
                 % Instantiate LogMsgGroup class named newName, process FMT data
                 obj.(newName) = LogMsgGroup(newType, newName, newLen, newFmt, newLabels);
```

Comment:
Adding "addprop(obj, newName);" immediately above the erroring line fixes it, I think. I tried to commit this change, and it appeared to work, but now doesn't show up? I hope I didn't mess anything up?

I was in a "detached HEAD" state on my local copy. Maybe that didn't work? Here's what I did, for reference:
I called git fetch, then checked out origin/slicing-patch, made the change, commited it, and then tried to push to origin/slicing-patch. It showed successful... but I don't see it here in github.